typescript/packages/node-http/src/router.ts

```diff
@@ -349,6 +378,9 @@ export class TempoRouter<TEnv> extends BaseRouter<IncomingMessage, TEnv, ServerR
 				message = e.message;
 				this.logger.error(message, undefined, e);
 			}
+			if (e instanceof Error && this.hooks !== undefined) {
+				await this.hooks.executeErrorHooks(undefined, e);
```

Comment:
I would normally remove the `context` param from `this.hooks.executeErrorHooks` to avoid passing undefined as an argument, but eventually we'll refactor it so some context is passed.